_data/changelogs/component-modal.yml

```diff
@@ -2,6 +2,12 @@ title: Modal
 type: component
 changelogURL:
 items:
+- date: NNNN-NN-NN
```

Comment:
**chore**: chore: Add tomorrow's date. Also adjusted spacing.

```suggestion
  - date: 2024-12-18
```

Reply:
Fixed indents in [5e84d0d](https://github.com/uswds/uswds-site/pull/2991/commits/5e84d0d35f2724af4ba9f451cad2bd84bfbca270)